mongodbatlas/organizations.go

```diff
@@ -21,18 +21,24 @@ import (
 )
 
 const (
-	orgsBasePath = "api/atlas/v1.0/orgs"
+	orgsBasePath       = "api/atlas/v1.0/orgs"
+	invitationBasePath = orgsBasePath + "/%s/invites"
 )
 
 // OrganizationsService provides access to the organization related functions in the Atlas API.
 //
 // See more: https://docs.atlas.mongodb.com/reference/api/organizations/
 type OrganizationsService interface {
 	List(context.Context, *OrganizationsListOptions) (*Organizations, *Response, error)
+	ListUnacceptedInvitations(context.Context, string, *InvitationOptions) (*[]Invitation, *Response, error)
```

Comment:
[nit]
```suggestion
	ListUnacceptedInvitations(context.Context, string, *InvitationOptions) ([]*Invitation, *Response, error)
```

Reply:
mh.. in this way I have an issue with `root := new([]Invitation)` which creates `*[]Invitation`. Any reasons why you are suggesting to create an array of pointers instead of a pointer to an array?